test/Libraries/RevitIntegrationTests/ElementBindingTests.cs

```diff
@@ -799,7 +733,8 @@ public void Rebinding_ReboundAdaptiveComponentsAreNotDeleted()
             NodeModel adaptiveCompNode = model.CurrentWorkspace.Nodes.Where(x => x.Name == "AdaptiveComponent.ByPoints").First();
 
             var doc = DocumentManager.Instance.CurrentDBDocument;
-            var familyInstances = Utils.AllElementsOfType<FamilyInstance>(doc);
+            var familyInstances = Utils.AllElementsOfType<FamilyInstance>(doc).Where(x=>x.Name.Contains("3PointAC_SquareTruss"));
```

Comment:
made this test a bit more conservative, it was not testing that rebinding was also working, only that elements were not deleted, now it tests both.